lein-ancient/src/leiningen/ancient/check.clj

```diff
@@ -46,3 +46,20 @@
   [opts]
   (exit-with-status
     (utils/call-on-profiles-files check-file! opts)))
+
+(utils/deftask check-stdin
+  {:docstring "Check artifacts in list(s) provided via stdin."
+   :exclude   [:interactive :print :no-tests]}
```

Comment:
Also I'm not sure what this thing is. I assuming it has something to do with `lein help ancient check-stdin`.

Feel free to update it accordingly.